### PR TITLE
feat: migrate to `lspconfig` and `nvim-cmp` 🎉 

### PR DESCRIPTION
**Pull request by numToStr**

### CODE REVIEW

**Comment by EpsilonKu** [comment]
Is this branch for 0.6?

**Comment by numToStr** [comment]
> Is this branch for 0.6?

I always use `nightly`. So yeah, you can say that :)

File: neovim/.config/nvim/ftplugin/graphql.lua [hunk 1]
```diff
@@ -1 +0,0 @@
-vim.bo.commentstring = '#%s'
```
Comment (numToStr) [inline neovim/.config/nvim/ftplugin/graphql.lua:1]: This got merged in `nvim-treesitter`. And FWIW I use `Comment.nvim` which has native support for `graphql`.

**Comment by numToStr** [comment]
- maybe add terraform ls
- fix keybinds.lua or revert it
- merge keybinds.lua and autocmd.lua into defaults.lua

**Pull request merged**